apturicovid/viewController/StatsVC.swift

```diff
@@ -50,7 +50,12 @@ class StatsVC: BaseViewController {
         statsCollectionView.refreshControl = refreshControl
         refreshControl.layer.zPosition = -1
         
-        NotificationCenter.default.addObserver(self, selector: #selector(preferredContentSizeChanged(_:)), name: UIContentSizeCategory.didChangeNotification, object: nil)
+        NotificationCenter.default.rx
+            .notification(UIContentSizeCategory.didChangeNotification)
+            .subscribe(onNext: { [weak self] (_) in
```

Comment:
Add .observeOn(MainScheduler.instance) so there will be no need to call DispatchQueue.main in preferredContentSizeChanged()